R/board_databricks.R

```diff
@@ -0,0 +1,397 @@
+#' Use a Databricks Volume as a board
+#'
+#' Pin data to a [Databricks Volume](https://docs.databricks.com/en/sql/language-manual/sql-ref-volumes.html)
+#'
+#' # Authentication
+#'
+#' `board_databricks()` searches for an authentication token in three different
+#' places, in this order:
+#' - 'DATABRICKS_TOKEN' environment variable
+#' - 'CONNECT_DATABRICKS_TOKEN' environment variable
+#' -  OAuth Databricks token inside the RStudio API
+#'
+#' In most cases, the authentication will be a Personal Authentication
+#' Token ('PAT') that is saved as the 'DATABRICKS_TOKEN' environment variable. 
+#' To obtain a 'PAT' see: [Databricks personal access token authentication](https://docs.databricks.com/en/dev-tools/auth/pat.html).
+#'
+#' # Details
+#'
+#' * The functions in pins do not create a new bucket.
```

Comment:
The docs here talk about a "bucket" but I don't see reference to a bucket in the Databricks Volume docs. Would it be better to say we will not create a folder?